ts/sagas/startup.ts

```diff
@@ -303,6 +305,8 @@ function* initializeApplicationSaga(): IterableIterator<Effect> {
     loadVisibleServicesRequestHandler,
     backendClient.getVisibleServices
   );
+  // Update number of instabug unread messages
+  yield takeEvery(updateInstabugUnreadMessages, updateInstabugBadgeSaga);
```

Comment:
This is right but you can include this in instabug saga.
Che the comment in _ts/sagas/instabug.ts_ (you could remove this from here)